funcx_web_service/routes/funcx.py

```diff
@@ -841,23 +843,27 @@ def register_endpoint_2(user: User, user_uuid: str):
     except Exception as e:
         app.logger.debug("Caught random error : {}".format(e))
         response = {'status': 'error',
-                    'reason': f'Caught error while registering endpoint {e}'}
-
-    try:
-        forwarder_ip = app.config['FORWARDER_IP']
-        response = register_with_hub(
-                f"http://{forwarder_ip}:8080", endpoint_uuid, endpoint_ip_addr)
-        app.logger.debug(f"Successfully registered {endpoint_uuid} with forwarder")
+                    'reason': f'Caught error while registering endpoint - {e}'}
```

Comment:
Instead of using response to carry the information forward, you could add an `else` block to the `try` statement. This might be simpler, also this is a very minor thing.

Reply:
After thinking about this a bit more, maybe it's best to just do `return jsonify(response)` whenever there is an error to avoid any funny behavior later. What do you think?

Reply:
This works fine.